.github/workflows/gatk-tests.yml

```diff
@@ -80,25 +103,25 @@ jobs:
           distribution: 'adopt'
           cache: gradle
 
-      - name: 'Grant execute permission for gradlew'
-        run: chmod +x gradlew
-
       - name: 'Compile with Gradle'
         run: |
           ./gradlew compileJava ; ./gradlew installDist
 
       #Google Cloud stuff
-      - id: 'auth'
+      - id: 'gcloud-auth'
+        if: needs.check-secrets.outputs.google-credentials == 'true'
```

Comment:
its so annoying that this is how you do this...

Reply:
I can have put the `${{ secrets.GOOGLE_CREDS == '' }}` everywhere instead but that felt more mistake prone.

Reply:
No, you did the right thing.  It's github that has failed us.